src/AppInstallerCommonCore/Manifest/ManifestCommon.cpp

```diff
@@ -417,6 +417,19 @@ namespace AppInstaller::Manifest
         return "Unknown"sv;
     }
 
+    std::string_view UnsupportedArgumentToString(UnsupportedArgumentEnum unsupportedArg)

```

Comment:
I've been trying to call all of these `ToString` to make it easier to find them.

Reply:
Removed this function as it is no longer needed.